api/v1/ytsaurus_types.go

```diff
@@ -228,7 +228,7 @@ type InstanceSpec struct {
 	Volumes      []corev1.Volume      `json:"volumes,omitempty"`
 	VolumeMounts []corev1.VolumeMount `json:"volumeMounts,omitempty"`
 	//+optional
-	ReadinessProbeParams  HealthcheckProbeParams          `json:"readinessProbeParams,omitempty"`
+	ReadinessProbeParams  *HealthcheckProbeParams         `json:"readinessProbeParams,omitempty"`
```

Comment:
Next time please do separate PR for independent issues, that could be merged ages ago.